src/services/passwordRecovery/hooks/index.js

```diff
@@ -1,12 +1,15 @@
 const local = require('@feathersjs/authentication-local');
 const { NotFound } = require('@feathersjs/errors');
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Configuration' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5697327)